modules/i18n/src/translations/si-LK/portals/console.ts

```diff
@@ -3754,6 +3768,34 @@ export const console: ConsoleNS = {
                             view: "දේශීය හිමිකම් බලන්න"
                         }
                     },
+                    sections: {
+                        manageAttributeMappings: {
+                            custom: {
+                                description: "අභිරුචි සිතියම්කරණය හරහා පරිශීලකයා පිළිබඳ තොරතුරු සන්නිවේදනය කරන්න.",
+                                heading: "අභිරුචි ගුණාංග සිතියම්ගත කිරීම"
+                            },
+                            description: "ඒපීඅයි හෝ ඔබේ යෙදුම් සමඟ අන්තර්ක්‍රියා කරන විට අස්ගාර්ඩියෝ හි ගුණාංග සිතියම්" +
```

Comment:
```suggestion
                            description: "API හෝ ඔබේ යෙදුම් සමඟ අන්තර්ක්‍රියා කරන විට WSO2 Identity Server හි ගුණාංග සිතියම්" +
```

Merging this temporally, need to replace "WSO2 Identity Server" value dynamically